GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -339,7 +339,8 @@ public static HotkeySettings[] CreateDefaultSettings()
                     Hk(FileViewer.Command.PreviousOccurrence, Keys.Alt | Keys.Left),
                     Hk(FileViewer.Command.StageLines, Keys.S),
                     Hk(FileViewer.Command.UnstageLines, Keys.U),
-                    Hk(FileViewer.Command.ResetLines, Keys.R)),
+                    Hk(FileViewer.Command.ResetLines, Keys.R),
+                    Hk(FileViewer.Command.IgnoreAllWhitespace, Keys.Shift | Keys.Control | Keys.W)),
```

Comment:
nit pick
```suggestion
                    Hk(FileViewer.Command.IgnoreAllWhitespace, Keys.Control | Keys.Shift | Keys.W)),
```

Reply:
Yes, done. (But I will not get used to it.)